src/wavecal.jl

```diff
@@ -1077,15 +1232,16 @@ function comb_exp_get_and_save_fpi_wavecal(
         #change peak_waves to be measured peak wavelengths (not expected)
         peak_waves .= expect_peak_waves .- resid_vec'
 
-        expparams0 = [cavity_size, m_offset]
```

Comment:
These feels a little hardcoded in. Are you trying to make the inferred parameters closer to the same scale in magnitue?